app/routes/me.tsx

```diff
@@ -61,64 +69,133 @@ export const action: ActionFunction = async ({request}) => {
+          />
+
+          <div className="flex flex-wrap items-baseline justify-between mb-4">
+            <Label htmlFor="discord-id">Discord</Label>
+            <p
+              id="discord-message"
+              className="dark:text-blueGray-500 text-gray-500 text-lg"
+            >
+              {user.discordId ? (
+                <a
+                  className="text-black dark:text-white"
+                  href={`https://discord.com/users/${user.discordId}`}
+                >
+                  connected
+                </a>
+              ) : (
+                <a className="text-black dark:text-white" href={authorizeURL}>
+                  Connect to Discord
+                </a>
+              )}
+            </p>
           </div>
-        ) : (
-          <>
-            <div>You wanna connect your account to discord?</div>
-            <a href={authorizeURL}>Connect my KCD account to Discord</a>
-          </>
-        )}
-      </div>
-      <div>
-        <details>
-          <summary>Login on another device:</summary>
-          <div>
-            Scan this QR code on the other device:
-            <img src={data.qrLoginCode} alt="Login QR Code" />
+
+          <Input
+            id="discord-id"
+            name="discord"
+            value={userInfo.discord?.username ?? user.discordId ?? ''}
+            placeholder="n/a"
+            readOnly
+            className="mb-12 lg:mb-20"
+            aria-describedby="discord-message"
+          />
+
+          <Button type="submit">Save changes</Button>
+        </div>
+
+        <div className="col-span-full lg:col-span-4 lg:col-start-8">
+          <Label className="mb-4" htmlFor="chosen-team">
+            Chosen team
+          </Label>
+
+          <input
+            className="sr-only"
+            type="radio"
+            name="team"
+            value={user.team}
+            checked
+            readOnly
+          />
+
+          <div className="relative col-span-full mb-3 bg-gray-100 dark:bg-gray-800 rounded-lg focus-within:outline-none ring-team-current ring-offset-team-current ring-offset-team-current ring-offset-4 ring-2 focus-within:ring-2 lg:col-span-4 lg:mb-0">
+            <span className="absolute left-9 top-9 text-team-current">
+              <CheckIcon />
+            </span>
+
+            <div className="block pb-12 pt-20 px-12 text-center">
+              <img
+                className="block mb-16"
+                src={TEAM_MAP[user.team].image.src}
+                alt={TEAM_MAP[user.team].image.alt}
+              />
+              <H6>{TEAM_MAP[user.team].label}</H6>
+            </div>
           </div>
-        </details>
-      </div>
-    </div>
+        </div>
+      </Grid>
+    </Form>
   )
+
+  // TODO: something... I'm not sure how to fit these into the design
+  //  <Form method="post" action="/me">
+  //    <input type="hidden" name="actionId" value="logout" />
+  //    <button type="submit">Logout</button>
+  //  </Form>
```

Comment:
That's not true. I got feedback from Gil, and the current position of the logout button is approved. I only still don't entirely know how to adjust it for narrow breakpoints.